src/main/java/com/volunteer/enterprise/dto/Company.java

```diff
@@ -3,9 +3,16 @@
 import lombok.Data;
 
 public @Data class Company {
-
     private int companyID;
     private String companyEmail;
     private String companyUser;
     private String companyPass;
+
+    public String toString() {
+        return "Company - " +
+                " ID: " + companyID +
+                " Email: " + companyEmail +
+                " User: " + companyUser +
+                " Password: " + companyPass;
+    }
```

Comment:
Implemented more readable toString() than what Lombok provides for `Company` and `Volunteer` DTOs.

Reply:
Good.